targets/TARGET_STM/TARGET_STM32L4/l4_retarget.c

```diff
@@ -50,8 +48,8 @@ extern uint32_t __mbed_krbs_start;
 void *__wrap__sbrk(int incr)
 {
     static uint32_t heap_ind = (uint32_t) &__mbed_sbrk_start;
-    uint32_t heap_ind_old = STM32L4_ALIGN_UP(heap_ind, STM32L4_HEAP_ALIGN);
-    uint32_t heap_ind_new = STM32L4_ALIGN_UP(heap_ind_old + incr, STM32L4_HEAP_ALIGN);
+    uint32_t heap_ind_old = heap_ind;
+    uint32_t heap_ind_new = heap_ind_old + incr;
 
     if (heap_ind_new > &__mbed_krbs_start) {
```

Comment:
Hi, could we take the opportunity of removing a compilation warning here ?
```
[Warning] l4_retarget.c@54,22: comparison between pointer and integer
[DEBUG] Return: 0
[DEBUG] Output: .\targets\TARGET_STM\TARGET_STM32L4\l4_retarget.c: In function '__wrap__sbrk':
[DEBUG] Output: .\targets\TARGET_STM\TARGET_STM32L4\l4_retarget.c:54:22: warning: comparison between pointer and integer
[DEBUG] Output:      if (heap_ind_new > &__mbed_krbs_start) {
[DEBUG] Output:                       ^
```

With
```
-    if (heap_ind_new > &__mbed_krbs_start) {
+    if (heap_ind_new > (uint32_t)&__mbed_krbs_start) {
```

Reply:
Thanks for catching this warning 💯